benchmarks/slab_detachment/doc/slab_detachment.md

```diff
@@ -6,96 +6,45 @@ Slab detachment (slab break-off) may occur in the final stages of subduction
+$\eta_0=4.75\times 10^{11}\text{ Pa . s}$. The mantle occupies the rest of the domain and
+has a constant viscosity $\eta_m=1\times 10^{21}\text{ Pa . s}$ and a density
+$\rho_m=3150\text{ kg/m}^3$. Viscosity is capped between $1\times10^{21}\text{ Pa . s}$
+and $1\times 10^{25} \text{ Pa . s}$. {numref}`fig:slab_detachment_evolution`
 shows the various fields and their evolution through time. As shown in
-(Schmalholz 2011; Glerum et al. 2018) the hanging slab necks, helped by the
+{cite:t}`schmalholz:2011,glerum:etal:2018` the hanging slab necks, helped by the
 localizing effect of the nonlinear rheology. Model results were shown to
-compare favorably to the results of (Schmalholz 2011) in (Glerum et al. 2018;
-Hillebrand et al. 2014) and the effect of viscosity and material averaging was
-explored in (Glerum et al. 2018).
-
-<div id="refs" class="references csl-bib-body hanging-indent">
-
-<div id="ref-garm18" class="csl-entry">
-
-Garzanti, E., G. Radeff, and M. G. Malus&agrave;. 2018. "Slab Breakoff:
-A Critical Appraisal of a Geological Theory as Applied in Space and
-Time." *Earth-Science Reviews* 177: 303--19.
-<https://doi.org/10.1016/j.earscirev.2017.11.012>.
-
-</div>
-
-<div id="ref-gltf18" class="csl-entry">
-
-Glerum, A., C. Thieulot, M. Fraters, C. Blom, and W. Spakman. 2018.
-"Nonlinear Viscoplasticity in <span class="smallcaps">ASPECT</span>:
-Benchmarking and Applications to Subduction." *Solid Earth* 9 (2):
-267--94. <https://doi.org/10.5194/se-9-267-2018>.
-
-</div>
-
-<div id="ref-hitg14" class="csl-entry">
-
-Hillebrand, B., C. Thieulot, T. Geenen, A. P. van den Berg, and W. Spakman.
-2014. "Using the Level Set Method in Geodynamical Modeling of
-Multi-Material Flows and Earth's Free Surface." *Solid Earth* 5
-(2): 1087--98. <https://doi.org/10.5194/se-5-1087-2014>.
-
-</div>
-
-<div id="ref-schm11" class="csl-entry">
-
-Schmalholz, S. M. 2011. "<span class="nocase">A simple analytical
-solution for slab detachment</span>."
-*Earth&nbsp;Planet.&nbsp;Sci.&nbsp;Lett.* 304: 45--54.
-
-</div>
-
-<div id="ref-vaal11" class="csl-entry">
-
-van Hunen, J., and M. B. Allen. 2011. "<span class="nocase">Continental
-collision and slab break-off: A comparison of 3-D numerical models with
-observations</span>." *Earth&nbsp;Planet.&nbsp;Sci.&nbsp;Lett.* 302:
-27--37.
-
-</div>
-
-<div id="ref-wosp00" class="csl-entry">
-
-Wortel, M. J. R., and W. Spakman. 2000. "<span class="nocase">Subduction
-and slab detachment in the Mediterranean-Carpathian region</span>."
-*Science* 290: 1910--17.
-
-</div>
+compare favorably to the results of {cite:t}`schmalholz:2011` in
+{cite:t}`glerum:etal:2018,hillebrand:etal:2014` and the effect of viscosity and material averaging was
+explored in {cite}`glerum:etal:2018`.
 
-</div>
+:::{admonition} TODO
+:class: error
 
-  [1]: #fig:slab_detachment_setup
-  [2]: #fig:slab_detachment_evolution
+Figure fig:slab_detachment_evolution missing in the manual pdf. Not sure what its supposed to look like.
+:::
```

Comment:
Found it. It was lost in #4585. Here is the latex code that should have been converted. Can you make a .md figure out of it?
```
\begin{figure}
\includegraphics[width=0.85\textwidth]{cookbooks/benchmarks/slab_detachment/doc/results.png}
\caption{\it Slab detachment benchmark: a,b) velocity and strain rate fields at $t=0$.
c,d,e) and f,g,h) time evolution of the viscosity and slab composition fields at $t=0, 6, 12\text{Myr}$.
\label{fig:slab_detachment_evolution}}
\end{figure}
```